ontology/hctl.html

```diff
@@ -110,6 +110,7 @@ <h2>Introduction</h2>
                 name of a previous definition given in a 
                 <code>schemaDefinitions</code> map must be used.</span><table class="def"><tbody><tr><td>Domain includes</td><td><code><a href="#AdditionalExpectedResponse">hctl:AdditionalExpectedResponse</a></code></td></tr></tbody></table></section>
 <section id="hasAnchor"><h4>hasAnchor</h4><p>IRI: <code>https://www.w3.org/2019/wot/hypermedia#hasAnchor</code></p><span>By default, the context, or anchor, of a link conveyed in the Link header field is the URL of the representation it is associated with, as defined in RFC7231, Section 3.1.4.1, and is serialized as a URI.</span><table class="def"><tbody><tr><td>Domain includes</td><td><code><a href="#Link">hctl:Link</a></code></td></tr></tbody></table></section>
+<section id="hasHreflang"><h4>hasHreflang</h4><p>IRI: <code>https://www.w3.org/2019/wot/hypermedia#hasHreflang</code></p><span>The hreflang attribute specifies the language for a linked document. The value of this attribute should conform to [[RFC5646]].</span><table class="def"><tbody><tr><td>Domain includes</td><td><code><a href="#Link">hctl:Link</a></code></td></tr></tbody></table></section>
```

Comment:
same comment here as above.

Nit: perhaps `the language for a linked document` => `the language of a linked document`?